blog/2024-08-19-gsoc-vertx-proxy-updates.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Upcoming Vert.x Http Proxy updates
+category: news
+authors:
+  - name: Zengyi Wang
+    github_id: wzy1935
+summary: >-
+    Introducing new features for Vert.x Http Proxy from GSoC 2024 project.
+---
+
+_This blog is also served as the final submission of the GSoC 2024 project._
+
+## Introduction
+
+[vertx-http-proxy](https://github.com/eclipse-vertx/vertx-http-proxy) is a component in the Vert.x ecosystem used 
+as an HTTP reverse proxy. Coming from Google Summer of Code 2024, some new features are implemented under this 
+project to make it more user-friendly. In this blog post, I introduce some features that are about to be included
+in the official release and how to use them, followed by reflections on the GSoC experience.
+
+<Alert warning>
+The updates mentioned in this post are not yet available from the maven repository; The actual results are subject 
+to the final provided API.
+</Alert>
+
+## Interceptors
+
```

Comment:
Can you please add a link to the corresponding PR here? And also do this for other sections?

Readers will then have an overview of the API and GSoC staff can see what you actually did during the project.